content/docs/setup/terraform/_index.md

```diff
@@ -6,52 +6,274 @@ weight = 30
+```
 
-If you do not wish to use Terraform, you can provision the servers manually with the following configurations.
+## Create a workflow
 
-#### Provisioner
+The next step is to combine both the hardware data and the template to create a workflow.
 
-- Plan: c3.small.x86 (or any plan that supports Layer 2)
-- OS: Ubuntu 18.04 LTS
-- After device is provisioned:
-  - Convert network type to Mixed/Hybrid
-  - Attach VLAN to interface eth1 (under Layer 2)
+- First, the workflow needs to know which template to execute. The Template ID you should use was returned by `tink template create` command executed above.
+- Second, the Workflow needs a target, defined by the hardware data. In this example, the target is identified by the MAC address you got back from the `terraform apply` command
 
-#### Worker
+Combine these two pieces of information and create the workflow with the `tink workflow create` command.
 
-- Facility: <same_as_provisioner>
-- Plan: c3.small.x86 (or any plan that supports Layer 2)
-- OS: Custom iPXE
-  - IPXE Script URL: https://boot.netboot.xyz
-  - Always/Persist PXE: true
-- After device is provisioned:
-  - Convert network type to Layer 2 (individual)
-  - Attach VLAN to interface eth0
-  - Same VLAN as provisioner
+```
+$ docker exec -i deploy_tink-cli_1 tink workflow create \
+    -t 75ab8483-6f42-42a9-a80d-a9f6196130df \
+    -r '{"device_1":"1c:34:da:5c:36:88"}'
+
+Created Workflow:  a8984b09-566d-47ba-b6c5-fbe482d8ad7f
+```
+
+The command returns a Workflow ID and if you are watching the logs, you will see:
+
+```
+tink-server_1  | {"level":"info","ts":1592936829.6773047,"caller":"grpc-server/workflow.go:63","msg":"done creating a new workflow","service":"github.com/tinkerbell/tink"}
+```
+
+## Check workflow status
+
+You can not ssh into the worker but you can use the CLI from the provisioner to
+validate if the workflow completed correctly using the `tink workflow events`
+command:
+
+```
+docker exec -i deploy_tink-cli_1 tink workflow events a8984b09-566d-47ba-b6c5-fbe482d8ad7f
+>
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+| WORKER ID                            | TASK NAME   | ACTION NAME | EXECUTION TIME | MESSAGE                         |      ACTION STATUS |
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+| ce2e62ed-826f-4485-a39f-a82bb74338e2 | hello world | hello_world |              0 | Started execution               | ACTION_IN_PROGRESS |
+| ce2e62ed-826f-4485-a39f-a82bb74338e2 | hello world | hello_world |              0 | Finished Execution Successfully |     ACTION_SUCCESS |
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+```
+
+---
+
+The `SOS` or `Out of bond` console provided by Packet is a very nice way to
```

Comment:
In the latest version of tink/deploy/terraform, this can be done with:
`ssh $(terraform output -json worker_sos | jq -r '.[0]')`